### PR TITLE
docs(log-drains): update appsignal and syslog tcp+tls examples and parameters

### LINKED ISSUE #1779: [LogDrains] Needs to be updated with the new features for appsignal and syslog (tcp+tls)

### PR DESCRIPTION
**Pull request by ipfaze**
This PR updates documentation about the new AppSignal compatibility and the token parameter for syslog (tcp+tls)
FIX #1779

### CODE REVIEW

**Review by curzolapierre — CHANGES_REQUESTED** [review]
Could you update the page metadata too, especially the `modified_at` field.

Otherwise LGTM, but this PR should be merge after the API changes

File: src/_posts/platform/app/2000-01-01-log-drain.md [hunk 1]
```diff
@@ -62,6 +63,17 @@ Parameters:
 More information on [ELK Stack on
 Scalingo]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %})
 
+#### AppSignal
+```bash
+scalingo --app my-app log-drains-add --type appsignal \
+  --token 123456789abcdef
+```
+Parameters:
+* `token`: API key of your application
```
Comment (curzolapierre) [inline src/_posts/platform/app/2000-01-01-log-drain.md:72]: To avoid any confusion.
```suggestion
* `token`: Appsignal's API key
```

**Comment by ipfaze** [comment]
Token's description and metadata `modified_at` are up to date

**Review by curzolapierre — APPROVED** [review]
LGTM

**Pull request merged**